lib/rules/no-multiple-assertions-wait-for.ts

```diff
@@ -0,0 +1,63 @@
+import { ESLintUtils, TSESTree } from '@typescript-eslint/experimental-utils'
+import { getDocsUrl } from '../utils'
+import { isBlockStatement, findClosestCallNode, isMemberExpression, isCallExpression, isIdentifier } from '../node-utils'
+
+export const RULE_NAME = 'no-multiple-expect-wait-for';
```

Comment:
The rulename should match the filename (and docs)

```suggestion
export const RULE_NAME = 'no-multiple-assertions-wait-for';
```